lib/onc_certification_g10_test_kit/smart_limited_app_group.rb

```diff
@@ -67,14 +79,13 @@ class SmartLimitedAppGroup < Inferno::TestGroup
           client_id: { locked: true },
           client_secret: { locked: true },
           url: { locked: true },
+          requested_scopes: { locked: true },
+          use_pkce: { locked: true },
+          pkce_code_challenge_method: { locked: true },
           code: { name: :limited_code },
           state: { name: :limited_state },
           patient_id: { name: :limited_patient_id },
           access_token: { name: :limited_access_token },
-          requested_scopes: {
-            name: :limited_requested_scopes,
```

Comment:
You can remove this from the preset. This input name also needs to be changed in lines 32, 155, and 166.

Reply:
Good point, not sure why I didn't run a global search on this.  Updated.  I also noticed that client_secret isn't marked as 'required', which it should be. Wouldn't make a difference but might as well have it be consistent.

Reply:
@Jammjammjamm I believe I addressed this, could you take another look?